src/scripts/main.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const img = document.querySelector('.spider');
+
+  if (!e.target.closest('.wall')) {
+    return;
+  }
+
+  img.style.clientTop = e.clientY + img.clientHeight / 2;
+  img.style.clientLeft = e.clientX + img.clientWidth / 2;
```

Comment:
The properties `clientTop` and `clientLeft` are incorrect for setting the position of an element. You likely meant to use `style.top` and `style.left` to set the position of the image. Additionally, ensure that the values are set with units, such as 'px'.